infrastructure/iac/types.go

```diff
@@ -40,3 +40,17 @@ type iacIssue struct {
 	Documentation  lsp.Uri        `json:"documentation"`
 	IacDescription iacDescription `json:"iacDescription"`
 }
+
+type IssueData struct {
+	Id            string   `json:"id"`
+	Title         string   `json:"title"`
+	Severity      string   `json:"severity"`
+	PublicId      string   `json:"publicId"`
+	Documentation string   `json:"documentation"`
+	LineNumber    int      `json:"lineNumber"`
+	Issue         string   `json:"issue"`
+	Impact        string   `json:"impact"`
+	Resolve       string   `json:"resolve:omitempty"`
```

Comment:
Does this represent "Remediation"?

Reply:
Yes, I'll rename it if it doesn't make sense to keep the same naming as IntelliJ